jetstream/tests/test_config.py

```diff
@@ -589,8 +596,8 @@ def test_segment_definitions(self, experiments):
             segments = ["regular_users_v3", "my_cool_segment"]
 
             [segments.my_cool_segment]
-            data_source = "my_cool_data_source"
-            select_expression = "{{agg_any('1')}}"
+            Data_source = "my_cool_data_source"
+            Select_Expression = "{{agg_any('1')}}"
```

Comment:
we want to be robust enough to allow stuff like this -> see above. lower casing in `config` module